src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```diff
@@ -512,7 +538,6 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
                     $vars = [
                         ...$vars,
                         'APPWRITE_FUNCTION_API_ENDPOINT' => $endpoint,
-                        'APPWRITE_FUNCTION_API_KEY' => API_KEY_DYNAMIC . '_' . $apiKey,
```

Comment:
does this remove dynamic api key from function?

Reply:
In code a little bit above, we add it. for functions only